rotkehlchen/db/upgrades/utils.py

```diff
@@ -0,0 +1,40 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.db.utils import update_table_schema
+from rotkehlchen.types import ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+
+
+def fix_address_book_duplications(write_cursor: 'DBCursor') -> None:
+    """
+    Ensure that there are no invalid addresses when upgrading the table
+    code taken from migration 17. It could happen that migration 17 wasn't
```

Comment:
```suggestion
    Ensure that there are no invalid addresses when upgrading the table.
    This code is moved here from migration 17. It could happen that migration 17 wasn't
```